scripts/generate_manifest

```diff
@@ -0,0 +1,15 @@
+#! python
+from argparse import ArgumentParser
+
+from mm_cataloguer.generate_manifest import Generate_Manifest
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(
+        description='Generate manifest of files requested from database')
+    parser.add_argument("-c", "--connection_string", help="DSN for modelmeta database")
+    parser.add_argument("-e", "--ensemble", nargs="+", default=["all"], 
+                        help="Ensembles from which files should be listed")
+    parser.add_argument("-s", "--since", help="Date after which files should be listed")
```

Comment:
If the user needs to provide a specific format (e.g. you are using "%Y-%m-%d" here), it's a necessity to specify that in the help string).

Better yet, the `python-dateutil` package, [which is already a requirement of this package](https://github.com/pacificclimate/modelmeta/blob/master/requirements.txt#L7), provides a [`parse()`](https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.parse) function that does its best to support multiple formats. I'd suggest using that here, and specify in the help that that's what does the parsing (so the user has some way of following the docs to determine what formats can be used).

And again, parse it in the nominal `main()` code here before passing it to `generate_manifest`.